assessments/EOCEP/templates/studentAssessmentEducationOrganizationAssociations.jsont

```diff
@@ -0,0 +1,15 @@
+ {
+  "educationOrganizationAssociationTypeDescriptor": "{{educationOrganizationAssociationTypeDescriptor}}",
+  "educationOrganizationReference": {
+    "educationOrganizationId": {{educationOrganizationId}}
```

Comment:
I realized in testing that this reference will also need a link:rel  in the edOrgRef, e.g.
```
{
  "educationOrganizationId": 1802,
  "link": {
    "rel": "LocalEducationAgency"
  }
}
```

This is needed for the warehouse to be able to pull out what kind of ed org it is. However, `link` requires an `href` in Ed-Fi (I think), and we don't have a value to supply to that, without reaching out to the ODS. This might be complicated. Could work for historic  years when we go around the ODS, though?

Reply:
@ryanaguilar @jayckaiser we have decided this is no longer a necessary change for the bundle, so please disregard. The bundle is approved and can be used for loading. (cc: @rlittle08)